spec/services/idv/actions/verify_document_status_action_spec.rb

```diff
@@ -32,13 +33,10 @@
 
       expect(subject).to receive(:verify_document_capture_session).
         and_return(verify_document_capture_session).at_least(:once)
-      expect(controller).to receive(:analytics).and_return(fake_analytics)
-      allow_any_instance_of(ApplicationController).
-        to receive(:analytics).and_return(fake_analytics)
```

Comment:
@stevegsa there was some redundant stubbing here... for me a big code smell/code alert is `allow_any_instance_of` ... so I worked to reduce that by updating our controller mock above